path_helper.py

```diff
@@ -0,0 +1,13 @@
+import pathlib
+
+
+class PathHelper:
+    @staticmethod
+    def to_absolute(relative: str) -> str:
+        return str(pathlib.Path(relative).resolve())
+
+    @staticmethod
+    def replace_extension(input_path: str, extension: str) -> str:
```

Comment:
If you left only one parameter, I'd call it `new: str` because we already have "replace extension" in the method name.

This way it'd be obvious that it's not an extension to replace with something, but a new extension.